### DISCUSSION TITLE
Why doesn't "fix common errors" recognize dialogue?

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5131] [General]
I had this issue with one other file, but it was apparently because the dashes weren't dashes, they just looked like them. In that other file, I just replaced all the fake dashes with real ones and that fixed it, but that's not working here. Those are definitely dashes, but SE thinks they're not dialogue.

![image](https://user-images.githubusercontent.com/82276985/122434988-985fe500-cf65-11eb-9be3-3a50b83fb5ae.png)

**Comment by niksedk** [comment]
Could you attach the subtitle - always make things much easier, thx :)

**Comment by DrReddP** [comment]
I tried
![image](https://user-images.githubusercontent.com/82276985/122436241-a82bf900-cf66-11eb-8468-eea99806c962.png)

**Comment by niksedk** [comment]
You can make a .zip file or rename to .txt

**Reply by DrReddP** [reply]
[Before removing HI.txt](https://github.com/SubtitleEdit/subtitleedit/files/6671888/Before.removing.HI.txt)
[After Removing HI.txt](https://github.com/SubtitleEdit/subtitleedit/files/6671889/After.Removing.HI.txt)

**Comment by niksedk** [comment]
The first line is missing a sentence terminator, like period, exclamation mark, or question mark.
